Jekyll Templating & SEO

samhita-alla opened the pull request:
Signed-off-by: samhita-alla <aallasamhita@gmail.com>

-  Copied changes from `master` to `development` branch (added navbar.html to include the navbar-related code)
-  Created sitemap.xml file
-  Added meta description
-  Added meta tag to enable Google Search Console Access
-  Jekyll SEO Tag
-  Modified Navbar

kumare3 commented on _config.yml:
```diff
@@ -1,8 +1,11 @@
 
+plugins:
+  - jekyll-seo-tag
+
```
We have to change links to docs

samhita-alla replied:
Done.

kumare3 commented on _includes/footer.html:
```diff
@@ -1,32 +1,72 @@
+          Ready To Take Flyte?
+        </div>
+        <a
+          href="https://docs.flyte.org/en/latest/getting_started/first_run.html"
```
This was a template right?

samhita-alla replied:
It's a template. I just removed the first `</div>` line from the file and included it in the `default.html` file. `footer`, `header`, and `navbar` HTML pages should only have their respective code blocks. If there are additional `div`-related tags, I've moved them to the `default.html` file.

The remaining modifications are due to the code prettifier.

kumare3 commented on _includes/footer.html:
```diff
@@ -1,32 +1,72 @@
+          <li class="my-2">
+            <a
+              class="link-white hover:border-purple-lightest"
+              href="https://flyte.readthedocs.io/en/latest/"
```
Let's just use docs.Flyte.org

samhita-alla replied:
Done.

kumare3 commented on sitemap.xml:
```diff
@@ -0,0 +1,189 @@
+  <url>
+    <loc>https://docs.flyte.org/projects/flytekit/en/latest/</loc>
+    <lastmod>2021-03-31</lastmod>
+    <changefreq>monthly</changefreq>
```
Can we make this daily?

samhita-alla replied:
Sure. I auto-generated the sitemap.xml file. So it's populated as monthly.

kumare3 reviewed (commented):
Couple questions- seems you are building the site but you removed all the templates- is that intentional?. I am ok with, just want to know

samhita-alla commented:
I've removed the `_site` directory as it isn't required to be pushed to GitHub. Here's an excerpt from the documentation:

> _site: This is where the generated site will be placed (by default) once Jekyll is done transforming it. It’s probably a good idea to add this to your .gitignore file. 


I've also removed the `destination: _site` line from the `_config.yml` file as the same value is set by default.
